uaclient/api/u/pro/attach/auto/start_retry_service/v1.py

```diff
@@ -0,0 +1,42 @@
+import logging
+
+from uaclient.api.api import APIEndpoint
+from uaclient.api.data_types import AdditionalInfo
+from uaclient.api.u.pro.attach.auto.full_auto_attach.v1 import (
+    FullAutoAttachOptions,
+)
+from uaclient.config import UAConfig
+from uaclient.data_types import DataObject
+from uaclient.services import retry_auto_attach
+
+StartRetryServiceOptions = FullAutoAttachOptions
+
+
+class StartRetryServiceResult(DataObject, AdditionalInfo):
+    pass
+
+
+def start_retry_service(
```

Comment:
Maybe I am missing something here, but why are we adding an API endpoint for the retry ?

Reply:
This will be useful for cloud-init to start the service, but maybe we don't need it in this feature. We can discuss further in the replacement PR